### PR TITLE
Rename user to people

### PR DESCRIPTION
**Pull request by PVince81**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "activity",
-  "description": "This application enables users to view a log of activities about them or about files.",
+  "description": "This application people users to view a log of activities about them or about files.",
```
Comment (artonge) [inline package.json:3]:
```suggestion
  "description": "This application enables people to view a log of activities about them or about files.",
```

**Review by artonge — CHANGES_REQUESTED** [review]

File: appinfo/info.xml [hunk 2]
```diff
@@ -2,13 +2,13 @@
 <info xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://apps.nextcloud.com/schema/apps/info.xsd">
 	<id>activity</id>
 	<name>Activity</name>
-	<summary>This application enables users to view a log of activities about them or about files.</summary>
+	<summary>This application enables people to view a log of activities about them or about files.</summary>
```
Comment (nickvergessen) [inline appinfo/info.xml:5]:
```suggestion
	<summary>This application enables people to view a log of activities about them, their files or other data.</summary>
```

File: appinfo/info.xml [hunk 3]
```diff
@@ -2,13 +2,13 @@
...
-		This application enables users to view actions related to their files in Nextcloud.
-		Once enabled, users will see a new icon “Activity” in their apps menu.
-		When clicked, a new page appears for users to track the activity related to files – from new files, to deleted files, move, rename, updates and shared activity.
-		The user can configure their individual activity settings in their personal menu.
-		This sets the type of activity to record, as well as whether to the user sees their own activities,
+		This application enables people to view actions related to their files in Nextcloud.
+		Once enabled, people will see a new icon “Activity” in their apps menu.
+		When clicked, a new page appears for people to track the activity related to files – from new files, to deleted files, move, rename, updates and shared activity.
+		A person can configure their individual activity settings in their personal menu.
+		This sets the type of activity to record, as well as whether the person sees their own activities,
```
Comment (nickvergessen) [inline appinfo/info.xml:11]: The whole description is so wrong, but let's keep it for now and fix it next time

**Review by artonge — APPROVED** [review]

**Review by nickvergessen — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Looks good!

**Comment by nickvergessen** [comment]
String freeze, can only be merged after 18th October

**Comment by nickvergessen** [comment]
Rebased and now it's time to merge

**Pull request merged**